src/tests/ftest/util/osa_utils.py

```diff
@@ -181,6 +181,19 @@ def verify_single_object(self):
         self.obj.close()
         self.container.close()
 
+    def add_cont(self, pool):
```

Comment:
Isn't this the same as `add_container` in test.py? i.e., calling `add_cont(my_pool)` is the same as calling `add_container(my_pool)`

Reply:
I noticed it after adding this method... Let me take a look at it... If it works, I can use add_container. Good suggestion.

Reply:
Done. Started using add_container.